pyccel/codegen/printing/ccode.py

```diff
@@ -1,6 +1,11 @@
 # coding: utf-8
 # pylint: disable=R0201
 
+from sympy.core import Tuple
```

Comment:
This is a new import but I can't see where it is used

Reply:
yes it need to be removed, it was used in the if statement.